beacon_node/beacon_chain/src/validator_monitor.rs

```diff
@@ -423,6 +473,33 @@ impl<T: EthSpec> ValidatorMonitor<T> {
                         );
                     }
                 }
+
+                // Indicates the number of sync committee signatures that made it into
+                // a sync aggregate.
+                if let Some(sync_committee) = &self.current_sync_committee {
+                    if sync_committee.contains(pubkey) {
```

Comment:
This is incorrect as the sync committee for the current state may not be equal to sync committee at `prev_epoch`